module/x/gravity/abci.go

```diff
@@ -58,16 +58,29 @@ func createSignerSetTxs(ctx sdk.Context, k keeper.Keeper) {
 	//	    that excludes him before he completely Unbonds.  Otherwise he will be slashed
 	// 3. If power change between validators of Current signer set and latest signer set request is > 5%
 	latestSignerSetTx := k.GetLatestSignerSetTx(ctx)
-	lastUnbondingHeight := k.GetLastUnbondingBlockHeight(ctx)
 	if latestSignerSetTx == nil {
 		k.CreateSignerSetTx(ctx)
 		return
 	}
+
+	lastUnbondingHeight := k.GetLastUnbondingBlockHeight(ctx)
+	blockHeight := uint64(ctx.BlockHeight())
 	powerDiff := types.EthereumSigners(k.CurrentSignerSet(ctx)).PowerDiff(latestSignerSetTx.Signers)
-	if (lastUnbondingHeight == uint64(ctx.BlockHeight())) || (powerDiff > 0.05) {
+
+	// TODO(levi) restore this!!!: createSignerSetTx := (lastUnbondingHeight == blockHeight) || (powerDiff > 0.05)
+	shouldCreateSignerSetTx := (lastUnbondingHeight == blockHeight) || (powerDiff > 0.02) // XXX(levi) deleteme and restore the prior line! this is just for debugging
```

Comment:
I lowered this because I don't think the test suite (start-testnet + test_runner) is doing its delegation math correctly. I need to validate that & will restore this to 5% before we merge this.  (Going to go mark this PR as a now DRAFT)